src/classes/OPP_OpportunityContactRoles_TDTM.cls

```diff
@@ -158,7 +166,7 @@ public without sharing class OPP_OpportunityContactRoles_TDTM extends TDTM_Runna
     ********************************************************************************************************/
     private static void updateHonoreeNotificationFields(list<Opportunity> listOpps, list<Opportunity> listOldOpps) {
         map<id,Contact> mapConIdContact = new map<id,Contact>();
-        list<String> listFields = new list<string>{'Honoree_Contact__c','Notification_Recipient_Contact__c'};
+        list<String> listFields = new list<string>{FIELD_NAME_HONOREE_CONTACT, FIELD_NAME_NOTIFICATION_RECIPIENT_CONTACT};
```

Comment:
@joy-sfdo I'm not sure this will work in a namespaced org. These _constants_ contain a non-namespace field name only, so once the `npsp` namespace is applied I'm pretty sure the code below (not displayed here - line 175+) will fail when it attempts to do a `get()` on the field name. Given how these constants are used (there's a 3rd one up there too), you may need to change everything to an SObject Field Token rather than a string unless you're confident that the namespace can be applied to the string without breaking any string comparisons. I'll leave the final implementation up to you.

Reply:
@force2b  -- I tested in a namespaced scratch org and the code ran as expected. Given that Strings were used originally, I'd like to leave this code as is to minimize changes. However, if something is uncovered during testing, I'm open to making the necessary changes.

Reply:
@joy-sfdo I see what you mean. It's really really strange. There's absolutely no valid reason this should work. The field string is not namespaced, yet I absolutely agree that it is not crashing. I tested this out heavily with debugging on and added some debug statements to see what's happening in the code. Here's an example showing two gets, one with a namespace and one without. They **both** work!
```
ERROR|>>>> NewOpp.npsp__Honoree_Contact__c == "003f000000z0SftAAE"
ERROR|>>>> NewOpp.Notification_Recipient_Contact__c == "003f000000z0SfvAAE"
```

However, I'm concerned that this is simply a core bug that allows us to get away with field get. I can see where this can easily fail if the customer has a custom field with the same field name. As a result, I think we must fix it in the code even if it's currently working.

Reply:
@force2b -- Adding the UTIL_Namespace method to the methods in OPP_OpportunityContactRoles_TDTM caused the code to stop working in a namespaced scratch org. I see your concern about it possibly being a core bug and impacting customers down the line. However, given that this code is already in production and customers are using it (Strings, the Constants are new), the risk of continuing with the code as is is lower than introducing a new element. Also, since adding the UTIL_Namespace code to the various methods using field get caused the code to stop working in a namespaced org, it will take time to investigate, debug, and rewrite the code which would mean stopping testing and possibly delaying the release of this feature. Therefore, we need to have a larger group discussion about modifying this code.

@mpusto, @alex-peralta -- Thoughts?

Reply:
I can't speak to it from a technical perspective, but is it possible to release and then change this next sprint to protect against the core issue? If it is working in both there will be no customer impact.

Reply:
It does technically work in our namespaced scratch org, but that is not representative of a package install in a customer org. There is still an open known issue that relates to this as well -- https://success.salesforce.com/issues_view?id=a1p30000000sY2FAAU. For example, a customer who happens to have their own `Honoree_Contact__c` field on the Opportunity object would almost certainly either get an error creating/changing Opportunities or the triggers would not create/change the proper OCR's.

Reply:
Is there a way to test against a representative org?

Reply:
@joy-sfdo pointed out that the actual base code was there before. I've personally bumped into this issue, so I'm setting up a full beta org to test against now. Given that the field names were hardcoded without namespaces previously (just in a different manner) I think that best case we leave it as is. Worse case, we can probably leave it as is and create a new GH issue for it as a bug to be fixed in a future release.

Reply:
Ok. Worked fine in a beta org even with 3 custom fields with identical names to the packaged ones. I'm baffled. The known issue is still there, and I was personally subject to this issue a few years back. Maybe core changed the functionality to always assume that the packaged fields are the ones to retrieve when using string field names? I guess it would be an issue for code that needed to actually retrieve the customer's field as opposed to the managed one, but that's not an issue for us to worry about. 

Let's leave the code here as is. I apologize for the fire drill.

Reply:
@force2b -- Sounds good. Thanks for calling out this item, the link to the Known Issue, and testing out in the beta org. 

I'll leave the code here as is.